ui/lib/components/CardTable/UnselectableRow.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { getTheme } from 'office-ui-fabric-react/lib/Styling'
+import {
+  DetailsRow,
+  IDetailsListProps,
+  IDetailsRowStyles,
+  IDetailsRowProps,
+} from 'office-ui-fabric-react'
+
+const theme = getTheme()
+
+export const createUnselectableRow = (
+  when: (props: IDetailsRowProps) => boolean
+) => {
+  const renderRow: IDetailsListProps['onRenderRow'] = (props) => {
+    if (!props) {
+      return null
+    }
+
+    const customStyles: Partial<IDetailsRowStyles> = {}
+    if (when(props)) {
+      customStyles.root = {
+        backgroundColor: theme.palette.neutralLighter,
```

Comment:
Is it possible to make it looks normally as other rows, but does not have hover backgrounds and uses a default cursor?

Reply:
We can't customize the cursor in selection area.